index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com/css?family=Inter">
+  <link rel="stylesheet" href="style.css">
+  <title>Daniel Canon's Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <div>
+        <ul class="shortcut-bar">
+          <li><a href="#projects">Portfolio</a></li>
+          <li><a href="#About-me">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+      <div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="container">
+      <div>
+        <h1>Hey there. I'm Daniel.</h1>
+        <p id="presentation">I'm a software developer</p>
+      </div>
+      <div>    
+        <p>I can help you build a product , feature or website Look through  some of my work and experience! <br> If you like what you see and have  a project you need coded, 
+        don't hesitate to contact me.</p>
+      </div>
```

Comment:
Kindly you should stick to the design as much as possible using the same (font, margin, padding, colors, etc.), I suggest you use the same font in the Figma design.

>### Your design
>
><img width="363" alt="Screenshot 2023-05-05 at 11 18 35 PM" src="https://user-images.githubusercontent.com/91237525/236596085-02348192-104c-4064-afd8-c6539d346ccb.png">
>
>### The Figma design
>
><img width="186" alt="Screenshot 2023-05-05 at 11 19 34 PM" src="https://user-images.githubusercontent.com/91237525/236596124-cc321299-4e0b-4c7f-8b42-ea4c64c7bb4c.png">
>
>### Kindly make sure you follow the Figma template and make sure you use the correct layout before asking for another review